projects/rocprofiler-sdk/source/lib/output/generateOTF2.cpp

```diff
@@ -28,6 +28,7 @@
 #include "lib/common/mpl.hpp"
 #include "lib/common/units.hpp"
 #include "lib/common/utility.hpp"
+#include "lib/rocprofiler-sdk/agent.hpp"
```

Comment:
This library is built before rocprofiler-sdk. Let's just define a local constexpr here.